loop/conftest.py

```diff
@@ -1,15 +1,79 @@
-# Configuration file for running our tests
+import ConfigParser
 import pytest
 
+from firefox_puppeteer import Puppeteer
+from marionette_driver.marionette import Marionette
+TIMEOUT = 20
+
 
 def pytest_addoption(parser):
+    """Options to be passed in at run time"""
     parser.addoption(
         "--env",
         action="store",
-        help="choose a test environment: staging or production"
+        help="choose a test environment: stage or prod"
+    )
+    parser.addoption(
+        "--bin",
+        default='/Applications/FirefoxBeta.app/Contents/MacOS/firefox-bin',
+        help='Path for Firefox binary'
```

Comment:
I'd also add an additional comment here in case the user missed it in the README:  "Path to Firefox binary (currently works on OSX only)"